.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+# this is just a config file for gitpod. Try it out at https://gitpod.io/#https://github.com/temporalio/samples-typescript/
+ports:
+  - port: 5432
+  - port: 7233
+  - port: 8088
+  - port: 9200
+tasks:
+  - name: Run Hello World Worker
+    init: |
+      cd helloworld
+      gp open README.md
+    command: |
+      gp sync-await server-up
+      go run /worker/main.go
```

Comment:
Can you confirm this works instead of `go run ./worker/main.go`? (I know, our READMEs are bad)

Reply:
agreed that it doesnt work and changed